Cargo.toml

```diff
@@ -27,3 +27,6 @@ panic = "abort"
 opt-level = 3
 codegen-units = 1
 lto = true
+
+[patch.crates-io]
+core2 = { git = "https://github.com/TheBlueMatt/core2", branch = "main" }
```

Comment:
Is this still `[DO NOT MERGE]`?

Reply:
Yea, depends on https://github.com/technocreatives/core2/pull/7. I *guess* we could just bump the CI MSRV version for no-std, but I don't see a reason to - gotta replace the `std::io` uses we have with `core2` anyway before we can ship no-std, so its not like we're in a rush.